src/utility_belt/sanitization.clj

```diff
@@ -0,0 +1,19 @@
+(ns utility-belt.sanitization
+  (:require
+    [clojure.string]
+    [utility-belt.validation]))
+
+(defn remove-spaces
```

Comment:
This is NIT, but maybe `remove-whitespace` is more precise?

Reply:
Since in internal use, will have to be fixed in 1.4 with deprecation in next minor release.